grafana/dashboards/cmode/snapmirror.json

```diff
@@ -882,15 +882,15 @@
       "pluginVersion": "8.1.8",
       "targets": [
         {
-          "expr": "avg (snapmirror_last_transfer_duration{source_cluster=~\"$SourceCluster\",source_volume=~\"$SourceVolume\",destination_volume=~\"$DestinationVolume\"}) by (relationship_id)",
+          "expr": "snapmirror_last_transfer_duration{source_cluster=~\"$SourceCluster\",source_volume=~\"$SourceVolume\",destination_volume=~\"$DestinationVolume\"}",
           "format": "table",
           "instant": true,
           "interval": "",
           "legendFormat": "",
           "refId": "C"
         },
         {
-          "expr": "avg(snapmirror_last_transfer_size{source_cluster=~\"$SourceCluster\",source_volume=~\"$SourceVolume\",destination_volume=~\"$DestinationVolume\"}) by (relationship_id)",
+          "expr": "snapmirror_last_transfer_size{source_cluster=~\"$SourceCluster\",source_volume=~\"$SourceVolume\",destination_volume=~\"$DestinationVolume\",relationship_id=~\"$TopTransferDataById\"}",
```

Comment:
it feels a bit surprising that the `Last Transfers` table is filtered by topk size. We should probably update the title of this panel to make that clear. Not sure folks will understand that the number of rows shown is a function of topk

Reply:
Changed to `Top $TopResources Last Transfers by Transfer Data Size`

Reply:
added filter for all columns.

For `all` option in topk,
Currently 500 is max value in topk. May be we provide highest value as some ~K, which gives all records.

Reply:
As `topk(100, snapmirror_labels)` function only allow k number as arg, I would be thinking this way:  
<img width="345" alt="image" src="https://user-images.githubusercontent.com/83282894/219332544-202eae0e-ad0b-4b96-af72-68d4be76945c.png">

Reply:
Tried this, But topk query would fail with this because not able to parse `|`.

<img width="677" alt="image" src="https://user-images.githubusercontent.com/83282894/219335050-aca7cc58-328b-47ed-826f-5345af2acbe5.png">

<img width="647" alt="image" src="https://user-images.githubusercontent.com/83282894/219335303-da5ec8cd-0f8a-4020-8cf8-94b71f683b0d.png">

<img width="1781" alt="image" src="https://user-images.githubusercontent.com/83282894/219335519-b580eb1b-e631-4571-b488-d97a435abe30.png">

Reply:
Missed the custom all value.... 
made the changes for `All`.